File: src/main/java/com/example/testservise/servise/TestService.java
package com.example.testservise.servise;

import com.example.testservise.domain.dto.TestDto;

import java.util.List;

public interface TestService {
    TestDto findByIdTest(Long id);
    List<TestDto> findAllTest();
    TestDto createTest(TestDto testDto);
    TestDto updateTest(TestDto testDto);
    boolean deleteTest(Long id);
}

File: src/main/java/com/example/testservise/servise/TestService.kt
package com.example.testservise.servise

import com.example.testservise.domain.entity.TestEntity

interface TestService {
    fun findByIdTest(id: Long): TestEntity?
    fun createTest(testEntity: TestEntity): TestEntity
    fun updateTest(testEntity: TestEntity): TestEntity?
    fun deleteTest(id: Long): Boolean
}